hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/ScmBlockDeletingServiceMetrics.java

```diff
@@ -272,6 +315,19 @@ public void getMetrics(MetricsCollector metricsCollector, boolean all) {
           .addGauge(DatanodeCommandDetails.BLOCKS_SENT_TO_DN_COMMAND,
           e.getValue().getBlocksSent());
     }
+
+    // add metrics for deleted block transaction summary
+    if (blockManager.getDeletedBlockLog().isTransactionSummarySupported()) {
```

Comment:
we do not need feature flag, to be added for cases having impact in rollback, like,
1. saving to db
2. communication to other service
Metrics are fine to be recorded

Reply:
We need a feature flag to mark a clear point that which deletion tx will be counted into summary, and others will be ignored.